app/packages/looker/src/lookers/abstract.ts

```diff
@@ -546,9 +552,9 @@ export abstract class AbstractLooker<
         if (buffer.detached) {
           // most likely sample is already being processed, skip update
           return;
-        } else {
-          arrayBuffers.push(buffer);
         }
+
+        arrayBuffers.push(buffer);
```

Comment:
Should we add a guard for undefined buffer?
```suggestion
        if (buffer) {
          arrayBuffers.push(buffer);
        }
```

Reply:
There is a falsey check higher up already. I will do a little more cleaning, though